### DISCUSSION TITLE
Add bookmark collections

### DISCUSSION THREAD

**Discussion by dreamy32** [dailydotdev/daily#1557] [Feature Requests]
I often bookmark post on daily dev, but I have no way to organize them. On instagram and facebook, I created bookmark collections such as Front-End, CSS, Product Management and more.

This allow me to keep track of my bookmarks for reading later.

This would be a game-changer.

_I saw a similar discussion, but it is from 2022._

**Comment by idoshamun** [comment]
Hi @dreamy32,

Let's keep the discussion here: https://github.com/dailydotdev/daily/discussions/520 even though it's an old one
